pyatlan/pkg/templates/default_configmap.jinja2

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ pkg.name }}
+data:
+  config: |-
+    {
+      "properties": {
+{%- for key, value in pkg.ui_config.properties.items() %}
+        "{{ key }}": {
+          "type": "{{ value.type_ }}",
+          "required": {{ value.required | tojson}},
+    {%- if value.possible_values %}
+          "enum": [ {%- for val in value.possible_values.keys() %}{{ " " }}"{{ val }}"{%- if not loop.last %},{%- endif %}{% endfor -%}{{ " ]" }},
+          "enumNames": [ {%- for val in value.possible_values.values() %}{{ " " }}"{{ val }}"{%- if not loop.last %},{%- endif %}{% endfor -%}{{ " ]" }},
```

Comment:
Just to confirm — `.values()` and `.keys()` return insertion order in both cases (they're lists, not sets)? Otherwise the keys and values will be skewed relative to each other and / or listed in the drop-down in a different order than what the user configured them in...

Reply:
Yes they are returned in insertion order.